### ISSUE TITLE
Display better error messages

### ISSUE THREAD

**Issue by syedbarimanjan** [opencv/cvat#7672]
### Actions before raising this issue

- [X] I searched the existing issues and did not find anything similar.
- [X] I read/searched [the docs](https://opencv.github.io/cvat/docs/)

### Is your feature request related to a problem? Please describe.

The error message displayed here doesnt help.
![Screenshot from 2024-03-24 23-49-53](https://github.com/opencv/cvat/assets/112156147/f12a29aa-e55f-44d3-b039-0372e36dab0e)

### Describe the solution you'd like

It would be better to display this error message in that dialog element.
![Screenshot from 2024-03-24 23-49-53](https://github.com/opencv/cvat/assets/112156147/cce22060-37e7-4754-a2b9-6d2f6221eadc)

**Comment by bsekachev** [comment]
Hello, 

- I agree that the current information is not really useful.
- I do not think users should see what you proposed, you are a developer, but the most of cloud users and on-premise annotators - are not. They do not need to see these technical details.

**Comment by syedbarimanjan** [comment]
> * not need to see these technical details.

Yes showing the error messages directly isnt a good idea instead, we could provide more helpful error messages. For instance, if the disk usage is causing an error, we should advise the user to free up some space or ensure that at least 10 percent of the disk is empty to run the software.
